src/main/java/seedu/mark/logic/commands/GotoCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.mark.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.mark.commons.core.Messages;
+import seedu.mark.commons.core.index.Index;
+import seedu.mark.logic.commands.exceptions.CommandException;
+import seedu.mark.logic.commands.results.CommandResult;
+import seedu.mark.logic.commands.results.GotoCommandResult;
+import seedu.mark.model.Model;
+import seedu.mark.model.bookmark.Bookmark;
+import seedu.mark.model.bookmark.Url;
+
+/**
+ * Opens a bookmark.
+ */
+public class GotoCommand extends Command {
+
+    public static final String COMMAND_WORD = "goto";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Opens the bookmark identified by the index number used in the displayed bookmark list.\n"
```

Comment:
```suggestion
            + ": Opens the bookmark identified by the index used in the displayed bookmark list.\n"
```

Reply:
I'm not sure which sounds better, but just a suggestion (then we can stick to the convention chosen)